backend/src/routes/api/namespaces/namespaceUtils.ts

```diff
@@ -60,11 +61,14 @@ export const applyNamespaceChange = async (
     throw createCustomError('Forbidden', "You don't have the access to update the namespace", 403);
   }
 
+  const disableServiceMesh = getDashboardConfig().spec.dashboardConfig.disableServiceMesh;
+
   let labels = {};
   switch (context) {
     case NamespaceApplicationCase.DSG_CREATION:
       labels = {
         'opendatahub.io/dashboard': 'true',
+        'opendatahub.io/service-mesh': String(!disableServiceMesh),
```

Comment:
There's an underlying issue with this logic, what happens with projects that are created with service-mesh disabled that are already running?
I saw the demo and the logic on the code, but I'm still unsure about hat specific behaviour.

Reply:
This is a very valid point @lucferbux, thanks for bringing this up. I think this deserves a deeper conversation as we would like to understand possible use cases here. Who is/will be in control of enabling service mesh? Is it the Dashboard? 

We might be missing some more orchestration logic, so we should figure out what are the paths here and come up with improvements.

But I think it has a broader scope, for example Kubeflow has env variable `USE_ISTIO=true` which will result in controllers creating `VirtualService`s. 

/cc @cfchase